e2e/getContractAddress.test.ts

```diff
@@ -0,0 +1,8 @@
+import { Client } from "../libs/client";
+
+describe("Verify client API get contract address", () => {
+  it("Shoud return the contract address if the client is running", async () => {
+    const client = new Client(process.env.APP_CLIENT_API_URL);
+    expect(await client.getContractAddress("Shield")).toBeTruthy();
+  });
+});
```

Comment:
This test exists in [nightfal_3/test/e2e/protocol/health-and-contracts.test.mjs](https://github.com/EYBlockchain/nightfall_3/blob/master/test/e2e/protocol/health-and-contracts.test.mjs#L35)